src/main/java/seedu/realodex/logic/commands/DeleteCommand.java

```diff
@@ -59,6 +59,7 @@ public DeleteCommand(Name targetName) {
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
+        assert model != null;
```

Comment:
Although assertions can be used to enforce preconditions (in this case model cannot be null), I believe the usage here is redundant because requireNonNull already does that for you.
Hence there is no way for the code to reach this line and so it would also be impossible to test for this

Reply:
Yes, I think remove